### ISSUE TITLE
Add Option to Group Actors / Deal with Large Graphs

### ISSUE THREAD

**Issue by elm-** [nraychaudhuri/akka-tree#13]
I guess the attached images explain the problem =) Looking for a way to group actors together. Maybe if the number of children goes above X just add one node that can be expanded further if required. 

![clipboard01](https://cloud.githubusercontent.com/assets/4736943/8146501/2a2a9b40-123b-11e5-9203-737d31545823.png)

**Comment by nraychaudhuri** [comment]
I was worried about this :) You know you can collapse tree branches by right click. Does that help? OTOH creating a single node when it goes more than X also sounds good.

**Comment by elm-** [comment]
Auto collapse maybe? I talking about models with tens of thousand of nodes, so display is not an option at all, maybe not even keeping it in memory. For larger graphs a query able backend is probably needed anyway.

**Comment by nraychaudhuri** [comment]
yeah in that scale you are right. We can build some knobs so that user can tweak it and we can apply that filter backend to reduce number of events sent from backend.
